bin/deploy.sh

```diff
@@ -1,45 +1,72 @@
 #!/usr/bin/env bash
```

Comment:
haven't worked with these scripts yet, just a general shell hygiene question: why no `set -euo pipefail`?

just in case:

1. `set -e`: Exit immediately if a command exits with a non-zero status.
2. `set -u`: Treat unset variables as an error when substituting.
3. `set -o pipefail`: The return value of a pipeline is the status of the last command to exit with a non-zero status, or zero if no command exited with a non-zero status.

Reply:
https://github.com/mento-protocol/mento-deployment/blob/9a4fecf4f8910bf8a5171b515b7e7237c5a477bb/bin/setup.sh#L1

We do it, but in the shared setup file, because our bash scripts have functions and are cool. 😂

Reply:
interesting, wasn't aware that these flags apply even to the calling script. was always naively assuming they're locally scoped per file. TIL something about bash :)

Reply:
It's true that they're scoped per script, but in this case we're sourcing the file, so we're basically executing all the commands there in the context of each script.